<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Selection Window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>830</width>
      <height>610</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>80</y>
       <width>411</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 1px solid rgb(51, 156, 255);
background-color: rgb(248, 248, 255);
border-radius : 30px;

</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="encrypt_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>191</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font: 13pt &quot;Calibri&quot;;
	border : 1px solid rgb(51, 156, 255);
	
	background-color: rgb(255, 255, 255);
	border-radius : 10px;
}
QPushButton::hover {
	font: 30pt &quot;Calibri&quot;;
	background-color: rgb(51, 156, 255);
	color : #ffffff;
}</string>
      </property>
      <property name="text">
       <string>Encrypt</string>
      </property>
     </widget>
     <widget class="QPushButton" name="decrypt_btn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>260</y>
        <width>191</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font: 13pt &quot;Calibri&quot;;
	border : 1px solid rgb(51, 156, 255);
	
	background-color: rgb(255, 255, 255);
	border-radius : 10px;
}
QPushButton::hover {
	font: 24pt &quot;Calibri&quot;;
	background-color: rgb(51, 156, 255);
	color : #ffffff;
}</string>
      </property>
      <property name="text">
       <string>Decrypt</string>
      </property>
     </widget>
     <widget class="QPushButton" name="info_btn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>330</y>
        <width>191</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font: 13pt &quot;Calibri&quot;;
	border : 1px solid rgb(51, 156, 255);
	
	background-color: rgb(255, 255, 255);
	border-radius : 10px;
}
QPushButton::hover {
	font: 16pt &quot;Calibri&quot;;
	background-color: rgb(51, 156, 255);
	color : #ffffff;
}</string>
      </property>
      <property name="text">
       <string>Info</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/Photoss/Photos/bitcoin (1).png);
border : 1px solid rgb(248, 248, 255);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>391</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border : 1px solid rgb(248, 248, 255);</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Select an operation to continue....&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>90</y>
       <width>411</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 1px solid rgb(51, 156, 255);
background-color: rgb(255, 255, 255);
border-radius : 30px;

</string>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
    <zorder>groupBox_6</zorder>
    <zorder>groupBox_5</zorder>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="SelectionWindow.qrc"/>
 </resources>
 <connections/>
</ui>
